src/cgame/cg_ents.c

```diff
@@ -1432,7 +1432,7 @@ static void CG_Missile(centity_t *cent)
 		}
 	}
 
-	if (cgs.sv_cheats && cent->currentState.clientNum == cg.predictedPlayerState.clientNum)
+	if ((cgs.sv_cheats || !cg.demoPlayback) && cent->currentState.clientNum == cg.predictedPlayerState.clientNum)
```

Comment:
This does the exact opposite :D it should be like this

```suggestion
	if ((cgs.sv_cheats || cg.demoPlayback) && cent->currentState.clientNum == cg.predictedPlayerState.clientNum)
```

Reply:
damn, I tested it though 😄

Reply:
it seemed working because I had sv_cheats 1 😄

Reply:
fixed